### PR TITLE
fixing actions spacing on modal

### PR DESCRIPTION
**Pull request by yale**
## Before
screen shot 2017-09-27 at 1 59 43 pm

## After
screen shot 2017-09-27 at 1 59 55 pm

### CODE REVIEW

File: src/components/Modal/style.scss [hunk 1]
```diff
@@ -70,3 +70,13 @@
 .content {
   @extend %layout--bounds;
 }
+
+.pageActions {
```
Comment (yale) [inline src/components/Modal/style.scss]: this style was being used in the `Modal` component but wasn't defined. I pulled this from the `Page` component.

File: src/components/Modal/style.scss [hunk 2]
```diff
@@ -70,3 +70,13 @@
...
+  display: flex;
+  *  {
```
Comment (wingyli) [inline src/components/Modal/style.scss]: minor: extra spacing here `* {`

Comment (rfarine) [reply src/components/Modal/style.scss]: FIX THE SPACING YALE

File: src/components/Modal/style.scss [hunk 3]
```diff
@@ -70,3 +70,15 @@
 .content {
   @extend %layout--bounds;
 }
+
+.pageActions {
+  display: flex;
+
+  * {
```
Comment (kangax) [inline src/components/Modal/style.scss]: You sure this is not too broad? Performance and future additions (of inside content) might suffer.

Comment (rfarine) [reply src/components/Modal/style.scss]: I was wondering about this too. I'm wondering if we'll ever have other content in the actions.

Comment (yale) [reply src/components/Modal/style.scss]: @nickstamas ^

**Pull request merged**